app/actionReorder/post.controller.js

```diff
@@ -0,0 +1,53 @@
+const { body } = require('express-validator')
+const { logger } = require('../../common/logging/logger')
+const { setActionPriorities } = require('../../common/data/sentencePlanningApi')
+const { catchAndReThrowError } = require('../../common/utils/util')
+
+const getActionRowKeys = bodyData => Object.keys(bodyData).filter(element => element.match(/^[0-9a-f-]{1,}$/g))
+
+const validationRules = () => {
+  return [
+    body()
+      // Check that all the priority index values are integers
+      .custom(value =>
+        getActionRowKeys(value).reduce((soFar, element) => soFar && Number.isInteger(Number(value[element])), true)
+      )
+      .withMessage('Priority values should all be integers')
```

Comment:
Might need a bit of content input here - I know it's unlikely that this will ever get triggered but probably should be something to help the user understand how to fix? Would the user even be able to do anything about it?